ratecompany/settings/tests.py

```diff
@@ -0,0 +1,24 @@
+from ratecompany.settings.base import *
+from ratecompany.settings.configs import *
+
+
+DEBUG = True
+
+ALLOWED_HOSTS = ["*"]
+
+SECRET_KEY = "some-khafan-key"
+
+EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+
+SMS_BACKEND = "CONSOLE"
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.contrib.gis.db.backends.postgis",
```

Comment:
Why you changed name and password?

Reply:
Excuse Me For That
first time that i was trying to work with docker-compose i used the the name and password(in django setting file) that is similar with environments in docker-compose file
after that i forgot to change them again for this PR.i'll make it same as others
sorry
https://github.com/jobguywork/backend/blob/2e8cb704dcabe61d36163bb336b938fa4d05a5b1/docker-compose.yml#L9-L12